core/src/client/node.rs

```diff
@@ -6,27 +6,28 @@ use alloy::primitives::{Address, Bytes, B256, U256};
 use alloy::rpc::types::{Filter, FilterChanges, Log, SyncInfo, SyncStatus};
 use eyre::{eyre, Result};
 
+use helios_common::{fork_schedule::ForkSchedule, network_spec::NetworkSpec, types::BlockTag};
+use helios_verifiable_api_client::VerifiableApiClient;
+
 use crate::consensus::Consensus;
 use crate::errors::ClientError;
 use crate::execution::evm::Evm;
 use crate::execution::rpc::http_rpc::HttpRpc;
 use crate::execution::state::State;
 use crate::execution::ExecutionClient;
-use crate::fork_schedule::ForkSchedule;
-use crate::network_spec::NetworkSpec;
 use crate::time::{SystemTime, UNIX_EPOCH};
-use crate::types::BlockTag;
 
 pub struct Node<N: NetworkSpec, C: Consensus<N::BlockResponse>> {
     pub consensus: C,
-    pub execution: Arc<ExecutionClient<N, HttpRpc<N>>>,
+    pub execution: Arc<ExecutionClient<N, HttpRpc<N>, VerifiableApiClient>>,
```

Comment:
Similar concern here. Can we have something that encompasses that it would use one or the other rather than both the HttpRpc and VerifiableApi client?

Reply:
`VerifiableApi` doesn't completely replace `HttpRpc` (as I said [here](https://github.com/a16z/helios/pull/505#issuecomment-2670692580)). 

So here the `ExecutionClient` is what encapsulates both `VerifiableApi` and `HttpRpc`.